starboard/common/string.h

```diff
@@ -101,6 +101,37 @@ static SB_C_FORCE_INLINE int strlcat(CHAR* dst, const CHAR* src, int dst_size) {
          dst_length;
 }
 
+// Splits a string on a char delimiter. Returns an empty vector if the delimiter
+// is not found
```

Comment:
Consider double checking if returning an empty vector when the delimiter is not found is a common practice.